src/index.ts

```diff
@@ -246,6 +255,24 @@ export class AcrolinxEndpoint {
     return getData<PlatformInformation>(this.getJsonFromPath('/api/v1/'));
   }
 
+  public async getAIEnabled(): Promise<AIEnabledInformation> {
+    return this.getJsonFromPath('/ai-service/api/v1/tenants/feature/ai-enabled');
+  }
+
+  public async getAIChatCompletion(prompt: string, count: number, issueInternalName: string): Promise<WriteResponse> {
```

Comment:
@JustinHorn We should also pass the accessToken to this function, the additionalHeaders are as the name suggest "additional" and not mandatory. For the ai-service authorization header is mandatory, we should accept it argument like rest of the checking api.

I would also see we get rid of additionalHeaders anyways, it is only a temporary thing for the tenant fed proxy, which will go away once we get the proper auth from other teams

Reply:
okay, passing the token 👍🏻